components/WidgetXTimeline.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import { useScriptTag } from '@vueuse/core'
+
+defineProps<{
+  src: string
+}>()
+
+useScriptTag('https://platform.twitter.com/widgets.js')
+</script>
+
+<template>
+  <div class="overflow-scroll rounded-[10px]">
+    <a class="twitter-timeline" :href="src">Loading</a>
```

Comment:
Remove class

Reply:
these classes come from the x widget script if we remove them the widget will lose format 🤔